### PR TITLE
adds topic description for alternatives to capitalist structures

### PR DESCRIPTION
**Pull request by Frijol**
@dcwalk did you want to co-lead this one with me, or are you full up with other topics?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,12 +47,16 @@ Coming soon! Readings should appear at least 1 month before the date of discussi
 
 ### Alternatives to Capitalist Structures
 
-Topic description coming soon!
+When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives: reduction of privacy, control from advertisers, and concentration of data ownership into a few corporations are all justified by capitalism's imperative for delivery of maximum value to shareholders.
```
Comment (dcwalk) [inline README.md]: list structure year a bit hard to follow -- is everything after the colon fears // challenges? Seems like there isn't agreement between items

Comment (Frijol) [reply README.md]: should have been a semicolon instead of a colon I think. Yes, the list is of fears & challenges– I think there's grammatical agreement, am I missing something? Feel free to take a crack at it

Comment (dcwalk) [inline README.md]:
```suggestion
When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives; the reduction of privacy, disproportional influence by advertisers, and concentrated ownership of data by a few corporations are all seemingly justified by the capitalist imperative to deliver maximum value to shareholders.
```

Comment (dcwalk) [reply README.md]: @Frijol suggested above

Comment (Frijol) [reply README.md]: thanks! (Also, cool feature! I didn't know you could do that)
